### ISSUE TITLE
[21.04] Crash when starting up from install USB - nautilus segfault

### ISSUE THREAD

**Issue by migxmac** [pop-os/beta#163]
**How did you upgrade to 21.04? (Fresh install / Upgrade)**

 Fresh install

 
**Related Application and/or Package Version (run `apt policy $PACKAGE NAME`):**

Installation does not complete. Issue seems to be in/around/related to nautilus

**Issue/Bug Description:**
When booting from an install USB for Pop OS 21.04, desktop and installation app come up for a few moments, then the desktop is replaced with the boot messages screen.
In dmesg: 
```
[  111.526954] kernel: nautilus[2574]: segfault at 20 ip 00007ff678843a7d sp 00007ffd9c12a528 error 4 in libgio-2.0.so.0.6800.0[7ff6787a2000+10b000]
[  111.526964] kernel: Code: 89 c6 48 8b 03 48 85 c0 74 05 48 39 30 74 0c 48 89 df e8 e6 42 f6 ff 85 c0 74 1a 48 8b 43 18 f6 40 30 01 74 38 48 8b 40 40 5b <48> 8b 40 20 c3 66 0f 1f 44 00 00 48 8d 15 e9 99 08 00 48 8d 35 0a
[  113.454527] kernel: EXT4-fs (sda2): mounted filesystem with ordered data mode. Opts: (null). Quota mode: none.
[  133.103723] kernel: rfkill: input handler enabled
```

**Steps to reproduce (if you know):**
Boot from install USB.
Wait

**Expected behavior:**
Boot from install USB
Step through installation app.
Complete installation of Pop OS! 21.04
Reboot
Enjoy!
[apport.log](https://github.com/pop-os/beta/files/6611831/apport.log)
[kern.log](https://github.com/pop-os/beta/files/6611834/kern.log)

**Other Notes:**
There is a file on the USB drive: writable/install-logs-2021-06-07.0/crash/_usr_bin_nautilus.999.crash
[usr_bin_nautilus.999.crash.zip](https://github.com/pop-os/beta/files/6611893/usr_bin_nautilus.999.crash.zip)

**Comment by bflanagin** [comment]
Thanks for trying the beta @migxmac. It goes without saying (though I'm saying it anyway) that we haven't seen this on our end. Could you give us more information about your computers hardware, and age.

**Comment by migxmac** [comment]
Sure thing:
Purchased in April of 2020
Hardware:  
- Motherboard: B450M PRO-VDH Max
- Processor: Ryzen 5 3600
- Graphics: Biostar Radeon RX 550 4GB GDDR5

I was able to get the beta installed via upgrade from 20.10.
I can attempt a reinstall from the beta USB drive if you would like more information.

**Comment by wheeljack73** [comment]
Also having this issue with a similar system.  Exactly the same symptom wise.
Computer assembled toward end of 2020.

I have the ISO on an SSD in a Zalman enclosure, which emulates cd/dvd for disk images.
Had no issues with this for previous popOS versions.

- Motherboard: Aorus Master X570 (Rev 1.2)
- Processor: Ryzen 9 5950X
- Graphics: Radeon RX 590
- Memory: 32 GB DDR4 (2 x 16GB)

Reinstalled 20.10, will attempt install again when a newer beta iso is released.  Or, if that fails again I will simply upgrade from 20.10 as opposed to doing a clean install.

**Comment by rbreaves** [comment]
Current build available for download right now, pop-os_21.04_amd64_intel_3.iso, also crashes on a core i5 from probably 2010-2011. Kinda old, but still. It simply says that the Gnome Display Manager has stopped running and not much else, a few other things say that finished or reached a target, but I am dumped on a terminal and can't do anything.

**Comment by leviport** [comment]
By chance, are any of these machines hooked up to ethernet? If so, does disconnecting ethernet allow them to boot the iso successfully?

**Comment by migxmac** [comment]
Can confirm:
  with ethernet disconnected I can boot up successfully. I even let it sit for about 10 minutes.
  I closed down the installer, launched Firefox, THEN connected ethernet. No Crash
  I launched the installer from the dock, no crash.

I rebooted with ethernet connected and I experienced the same issue when booting with the USB installer.

I disconnected ethernet, rebooted into the installer and the system did not crash.
